zds/tutorialv2/tests/tests_utils.py

```diff
@@ -589,12 +583,5 @@ def test_image_with_non_ascii_chars(self):
         self.assertTrue(PublishedContent.objects.filter(content_id=article.pk).exists())
 
     def tearDown(self):
-        if os.path.isdir(overridden_zds_app['content']['repo_private_path']):
-            shutil.rmtree(overridden_zds_app['content']['repo_private_path'])
-        if os.path.isdir(overridden_zds_app['content']['repo_public_path']):
-            shutil.rmtree(overridden_zds_app['content']['repo_public_path'])
-        if os.path.isdir(settings.MEDIA_ROOT):
-            shutil.rmtree(settings.MEDIA_ROOT)
-        if os.path.isdir(overridden_zds_app['content']['extra_content_watchdog_dir']):
-            shutil.rmtree(overridden_zds_app['content']['extra_content_watchdog_dir'])
+        super().tearDown()
```

Comment:
Si je ne dis pas de bêtises, ceci va casser si la classe du test hérite de plus d’un mixin qui possède un attribut `tearDown`, non ?

Reply:
Non, s'il y a plus d'un tearDown, il exécutera les teardown de toutes les classes : https://stackoverflow.com/questions/24166225/python3-behaviour-of-super-on-multi-inheritance